windows/wsl_distro_install_uninstall/install_wsl.yml

```diff
@@ -0,0 +1,51 @@
+# Copyright 2022 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Description:
+# Install inbox version of WSL
+
+# Install WSL and distribution
+- include_tasks: ../utils/win_execute_cmd.yml
+  vars:
+    win_powershell_cmd: "wsl --install --inbox -d {{ wsl_distribution_name }} | Out-File -FilePath {{ wsl_file_path_win }}\\wsl_install.txt"
+    win_execute_cmd_ignore_error: true
+
+- include_tasks: wsl_convert_binary_to_txt.yml
+  vars:
+    source_path: "{{ wsl_file_path_win }}\\wsl_install.txt"
+    file_name: "wsl_install.txt"
+
+- name: "Check WSL installation command result"
+  ansible.builtin.fail:
+    msg: "WSL installation failed. Please check log file wsl_install.txt."
+  when: win_powershell_cmd_output.rc != 0
+
+- include_tasks: ../utils/win_shutdown_restart.yml
+  vars:
+    set_win_power_state: "restart"
+
+# Set wsl default version to 2 and check the WSL version
+- include_tasks: ../utils/win_execute_cmd.yml
+  vars:
+    win_powershell_cmd: "wsl --set-default-version 2"
+    win_execute_cmd_ignore_error: true
```

Comment:
Here if set 'win_execute_cmd_ignore_error' to true, may miss the task failure issue.

Reply:
If this task is failed then no need to go on.

Reply:
Ok. remove the "ignore error"